tests/Unit/Evolution/Systems/CurvedScalarWave/Test_UpwindPenaltyCorrection.cpp

```diff
@@ -169,7 +169,7 @@ void test_upwind_flux_random() noexcept {
       CurvedScalarWave::CurvedScalarWave_detail::weight_char_fields<Dim>(
           char_fields_int, char_speeds_minus_one, char_fields_ext,
           char_speeds_minus_one);
-  CHECK(weighted_char_fields_minus_one == -1. * char_fields_ext);
+  CHECK_VARIABLES_APPROX(-1. * char_fields_ext, weighted_char_fields_minus_one);
```

Comment:
The changes in this file should probably be in the other commit, but I think it's fine to leave as-is, doesn't really matter